contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,29 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of problematic language from the documentation, use the following terminology when referring to OpenShift control plane or compute nodes:
+
+[options="header"]
+|===
+|Branch |Control plane node reference |Compute node reference
+
+|`main` and `enterprise-4.9`
+|Control plane node
+|Compute node
+
+|`enterprise-4.8` and earlier
+|Control plane (also known as master) node
```

Comment:
We already had the terms "control plane node" and "worker node" in pre-4.8 content. Can we clarify that if it already existed as such, that we don't have to change it to add the aka part?

Reply:
I think this comment has been resolved [here](https://github.com/openshift/openshift-docs/pull/35110#discussion_r683484160).